pkg/agent/openflow/framework.go

```diff
@@ -135,7 +137,16 @@ var tableOrderCache = make(map[binding.PipelineID][]*Table)
 type Option func(*Table)
 
 var defaultDrop = func(table *Table) {
-	table.missAction = binding.TableMissActionDrop
+	table.missAction = binding.MissAction{
+		ActionType: binding.TableMissActionDrop,
+	}
+}
+
+var defaultGoToSwitching = func(table *Table) {
+	table.missAction = binding.MissAction{
+		ActionType: binding.TableMissActionGoToStage,
+		Value:      uint8(stageSwitching),
+	}
```

Comment:
I feel we should add a new mechanism in the PR. It didn't have the ability to define the next table is to ensure every packet will go through the whole pipeline unless there is a special case. It may be controversial whether the design still makes sense but I think this PR is not a right place to make the change. You can just install a low priority flow to resubmit packets in QoS table to switching stage.

Reply:
Make sense. Done.